Schema/V1/no.ks.fiks.politisk.behandling.v1.felles.vedtak.schema.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "no.ks.fiks.politisk.behandling.v1.felles.vedtak.schema.json",
```

Comment:
Kanskje denne skulle vært gjort enda mer generell siden samme innhold vil gjelde også for saksbeskrivelse i utvalgssak.send.

Reply:
Ja, jeg har gjort en forandring nå og kalt felles komponenten for "beskrivelse" i stedet og bruker den da for vedtak, forslagTilVedtak og saksBeskrivelse.